### DISCUSSION TITLE
Welcome to PDFjet Discussions!

### DISCUSSION THREAD

**Discussion by edragoev1** [edragoev1/pdfjet#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by soster** [comment]
Happy to see some action here again!

I am Stefan and I am using pdfjet for my Android App Handwrite Pro. I am happy to help improving pdfjet as I already did in the past.

**Reply by edragoev1** [reply]
Hi Stefan,

Welcome to the forum. Thanks again for the code you provided in the past!

The current focus is improving the code examples and the documentation. Of course suggestions about improvements and additional functionality are always welcome. 

We would also like to hear about users of the Swift and Go versions.
